Yosemite/YosemiteTests/Stores/ShippingLabelStoreTests.swift

```diff
@@ -38,6 +38,130 @@ final class ShippingLabelStoreTests: XCTestCase {
+                                                                       address1: "No 70 RA St",
+                                                                       address2: "",
+                                                                       city: "Taipei",
+                                                                       postcode: "100")
+        let expectedRefund = Yosemite.ShippingLabelRefund(dateRequested: Date(timeIntervalSince1970: 1603716266.809), status: .pending)
+        let expectedShippingLabel = Yosemite.ShippingLabel(siteID: sampleSiteID,
+                                                           orderID: orderID,
+                                                           shippingLabelID: 1149,
+                                                           carrierID: "usps",
+                                                           dateCreated: Date(timeIntervalSince1970: 1603716274.809),
+                                                           packageName: "box",
+                                                           rate: 58.81,
+                                                           currency: "USD",
+                                                           trackingNumber: "CM199912222US",
+                                                           serviceName: "USPS - Priority Mail International",
+                                                           refundableAmount: 58.81,
+                                                           status: .purchased,
+                                                           refund: expectedRefund,
+                                                           originAddress: expectedOriginAddress,
+                                                           destinationAddress: expectedDestinationAddress,
+                                                           productIDs: [3013],
+                                                           productNames: ["Password protected!"])
+        let expectedSettings = Yosemite.ShippingLabelSettings(siteID: sampleSiteID, orderID: orderID, paperSize: .letter)
+        let expectedResponse = OrderShippingLabelListResponse(shippingLabels: [expectedShippingLabel], settings: expectedSettings)
+        remote.whenLoadingShippingLabels(siteID: sampleSiteID, orderID: orderID, thenReturn: .success(expectedResponse))
+        let store = ShippingLabelStore(dispatcher: dispatcher, storageManager: storageManager, network: network, remote: remote)
+
+        insertOrder(siteID: sampleSiteID, orderID: orderID)
+
+        // When
+        let result: Result<Void, Error> = try waitFor { promise in
+            let action = ShippingLabelAction.synchronizeShippingLabels(siteID: self.sampleSiteID, orderID: orderID) { result in
+                promise(result)
+            }
+            store.onAction(action)
+        }
+
+        // Then
+        XCTAssertNoThrow(try XCTUnwrap(result.get()))
```

Comment:
Using `isSuccess` is probably more understandable here. Especially since `result.get()` return `Void`. 

```suggestion
        XCTAssertTrue(result.isSuccess)
```

Reply:
Yup! Updated here: https://github.com/woocommerce/woocommerce-ios/pull/3176/commits/344ec72aaff55eb55026eacd893be18dbc55aa2d